collector/utils/utils.go

```diff
@@ -25,3 +30,14 @@ func SafeDereference[T any](s ...*T) []T {
 	}
 	return resolved
 }
+
+func SafeBytesToString(b []byte) string {
```

Comment:
A function description or at least a reference to the GitHub issue would helpful.

Reply:
Done.